src/main/java/no/nav/folketrygdloven/beregningsgrunnlag/regelmodell/resultat/Konstanter.java

```diff
@@ -0,0 +1,83 @@
+package no.nav.folketrygdloven.beregningsgrunnlag.regelmodell.resultat;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+import no.nav.folketrygdloven.beregningsgrunnlag.Grunnbeløp;
+
+/**
+ * Samling av ulike konstanter og toggler som brukes i reglene
+ */
+public class Konstanter {
```

Comment:
Er noe misvisende at den heter konstanter men man likevel har settere i klassen.

Reply:
Ja, det er for å tillate at man enkelt skal kunne legge til konstanter til eit eksisterende beregningsgrunnlag. Kan gjere settere package private